diff -uNr pygtk-2.3.91-old/gtk/gtk.defs pygtk-2.3.91/gtk/gtk.defs
--- pygtk-2.3.91-old/gtk/gtk.defs	2004-05-03 14:58:04.884287504 +0200
+++ pygtk-2.3.91/gtk/gtk.defs	2004-05-03 15:00:56.358219504 +0200
@@ -16798,7 +16798,7 @@
   (c-name "gtk_tree_model_filter_new")
   (return-type "GtkTreeModel*")
   (parameters
-    '("GtkTreePath*" "root")
+    '("GtkTreePath*" "root" (null-ok) (DEFAULT "NULL"))
   )
 )
 
