--- pygtk-2.24.0/gtk/Makefile.am.orig	2022-03-04 21:45:35.749117575 +0100
+++ pygtk-2.24.0/gtk/Makefile.am	2022-03-04 21:48:25.118200024 +0100
@@ -139,7 +139,7 @@ GTK_DEFS += gtk-2.24.defs
 GTK_TYPES_DEFS += gtk-2.24-types.defs
 endif
 
-CREATEDEFS = $(PYTHON) $(CODEGENDIR)/createdefs.py
+CREATEDEFS = $(PYTHON) $(CODEGENDIR)/createdefs.pyc
 
 GDK_OVERRIDES = \
 	gdk.override 		\
@@ -290,7 +290,7 @@ extra_codegen_args += --register $(PYGOB
 endif
 
 .defs.c: 
-	($(PYTHON) $(CODEGENDIR)/codegen.py \
+	($(PYTHON) $(CODEGENDIR)/codegen.pyc \
 	   $(PYGTK_CODEGEN_DEFINES) \
 	    -I $(srcdir) \
 	    --py_ssize_t-clean $(extra_codegen_args) \
diff -urN pygtk-2.14.0/Makefile.am pygtk-2.14.0.new/Makefile.am
--- pygtk-2.14.0/Makefile.am	2009-01-27 18:55:44.000000000 +0100
+++ pygtk-2.14.0.new/Makefile.am	2009-02-16 11:04:10.000000000 +0100
@@ -110,7 +110,7 @@
 # code generator
 .defs.c:
 	(cd $(srcdir)\
-	 && $(PYTHON) $(CODEGENDIR)/codegen.py \
+	 && $(PYTHON) $(CODEGENDIR)/codegen.pyc \
 	    --py_ssize_t-clean \
 	    --register pango-types.defs \
 	    --register atk-types.defs \
