diff -urN pygtk-2.14.0/gtk/Makefile.am pygtk-2.14.0.new/gtk/Makefile.am
--- pygtk-2.14.0/gtk/Makefile.am	2009-01-27 18:54:50.000000000 +0100
+++ pygtk-2.14.0.new/gtk/Makefile.am	2009-02-16 11:04:37.000000000 +0100
@@ -81,7 +81,7 @@
 GTKUNIXPRINT_DEFS += gtkunixprint-2.14.defs
 endif
 
-CREATEDEFS = $(PYTHON) $(CODEGENDIR)/createdefs.py
+CREATEDEFS = $(PYTHON) $(CODEGENDIR)/createdefs.pyc
 
 GDK_OVERRIDES = \
 	gdk.override 		\
@@ -217,7 +217,7 @@
 
 
 .defs.c: 
-	($(PYTHON) $(CODEGENDIR)/codegen.py \
+	($(PYTHON) $(CODEGENDIR)/codegen.pyc \
 	   $(PYGTK_CODEGEN_DEFINES) \
 	    -I $(srcdir) \
 	    --py_ssize_t-clean \
diff -urN pygtk-2.14.0/Makefile.am pygtk-2.14.0.new/Makefile.am
--- pygtk-2.14.0/Makefile.am	2009-01-27 18:55:44.000000000 +0100
+++ pygtk-2.14.0.new/Makefile.am	2009-02-16 11:04:10.000000000 +0100
@@ -110,7 +110,7 @@
 # code generator
 .defs.c:
 	(cd $(srcdir)\
-	 && $(PYTHON) $(CODEGENDIR)/codegen.py \
+	 && $(PYTHON) $(CODEGENDIR)/codegen.pyc \
 	    --py_ssize_t-clean \
 	    --register pango-types.defs \
 	    --register atk-types.defs \
