--- pygtk-1.99.13/codegen/pygtk-codegen-2.0.in.wiget	Mon Sep  9 13:05:36 2002
+++ pygtk-1.99.13/codegen/pygtk-codegen-2.0.in	Mon Sep  9 13:07:43 2002
@@ -7,4 +7,9 @@
 PYTHONPATH=$codegendir
 export PYTHONPATH
 
-exec @PYTHON@ $codegendir/codegen.py "$@"
+if [ -f $codegendir/codegen.pyc ] ; then
+	codegenexe="$codegendir/codegen.pyc"
+else
+	codegenexe="$codegendir/codegen.py"
+fi
+exec @PYTHON@ $codegenexe "$@"
